scale-tidb-using-tiup.md

```diff
@@ -399,9 +399,14 @@ In special cases (such as when a node needs to be forcibly taken down), or if th
         
 3. Wait for the store of the TiFlash node to disappear or for the `state_name` to become `Tombstone` before you stop the TiFlash process.
 
-    If, after waiting for a long time, the node still fails to disappear or the `state_name` fails to become `Tombstone`, consider using the following command to force the node out of the cluster.
-
-    **Note that the command will directly discard the replicas on the TiFlash node, which might cause the query to fail.**
+    If, after waiting for a long time, the node still fails to disappear or the `state_name` fails to become `Tombstone`, consider using the following command to force the node out of the cluster (use with caution).
+
+    > **Warning:**
+    > 
+    > - The following command directly discards the replicas on the TiFlash node, which might cause the query to fail.
+    > - Once used this command, do not use PD Control to delete the nodes from the meta-information of PD. Otherwise, the information of Regions in the cluster will be inconsistent.
+    > - You can only use the following command to the TiFlash node. Do not use this command to force any TiKV node out of the cluster. Otherwise, it will cause data loss.
```

Comment:
```suggestion
    > - The following command is only applicable to TiFlash nodes. Do not use this command to force any TiKV node out of the cluster. Otherwise, data loss occurs.
```